src/if.js

```diff
@@ -12,6 +12,7 @@ import {IfCore} from './if-core';
 export class If extends IfCore {
   @bindable({ primaryProperty: true }) condition: any;
   @bindable swapOrder: "before"|"with"|"after";
+  @bindable cache: boolean = true;
```

Comment:
I think if the value can potentially be string, we should be a more correct about the type here `boolean | string`